src/main/java/org/apache/mesos/hdfs/config/SchedulerConf.java

```diff
@@ -128,16 +128,15 @@ public double getTaskCpus(String taskName) {
   }
 
   public int getJournalNodeCount() {
-    return getConf().getInt("mesos.hdfs.journalnode.count", 1);
+    return getConf().getInt("mesos.hdfs.journalnode.count", 3);
   }
 
-  public String getClusterName() {
-    return getConf().get("mesos.hdfs.cluster.name", "mesos-ha");
+  public String getFrameworkName() {
+    return getConf().get("mesos.hdfs.framework.name", "hdfs");
   }
 
-  // TODO(elingg) This needs to be increased.
   public long getFailoverTimeout() {
-    return getConf().getLong("mesos.failover.timeout.sec", 0);
+    return getConf().getLong("mesos.failover.timeout.sec", 604800);
```

Comment:
Yup, I also discussed with Connor separately actually... I'm going to put 1
year. Let me know if there are any objections.

On Mon, Mar 9, 2015 at 2:15 PM, Adam <notifications@github.com> wrote:

> In src/main/java/org/apache/mesos/hdfs/config/SchedulerConf.java
> <https://github.com/mesosphere/hdfs/pull/72#discussion_r26077101>:
>
> >    public long getFailoverTimeout() {
> > -    return getConf().getLong("mesos.failover.timeout.sec", 0);
> > +    return getConf().getLong("mesos.failover.timeout.sec", 604800);
>
> Sure, but Marathon is stateless. You'd rather not stop your NN/DN tasks
> and lose access to your data just because somebody forgot to restart the
> scheduler for a week. As Connor mentioned at the Hackweek, there are really
> only two valid failoverTimeout values: '0' and 'as long as possible'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/hdfs/pull/72/files#r26077101>.
>